src/wallet/test/coinselector_tests.cpp

```diff
@@ -123,6 +123,7 @@ BOOST_AUTO_TEST_CASE(bnb_search_test)
 {
 
     LOCK(testWallet.cs_wallet);
+    testWallet.SetupLegacyScriptPubKeyMan();
```

Comment:
In commit "Implement GetKeyOrigin in LegacyScriptPubKeyMan" (b893f0d053f1b5be1229d4fb77ced46b222f9993)

It's unclear why this change is in this commit. Is it supposed to be part of the next one
93322bf756fd8991fb445e327c878e2695fe9ca9?

Reply:
Yeah. Moved it